wgsl/index.bs

```diff
@@ -261,9 +261,51 @@ that run on the GPU.
 
 <div class='example wgsl global-scope'>
   <xmp highlight='rust'>
+    // A fragment shader which lights textured geometry with point lights.
+
+    // Lights from a storage buffer binding.
+    struct PointLight {
+      position : vec3f,
+      color : vec3f,
+    }
+
+    struct LightStorage {
+      pointCount : u32,
+      point : array&lt;Light&gt;,
```

Comment:
The Treesitter parser will check this example.  These &lt; and &gt; should be ordinary < and >